moonstone/utils/taxonomy.py

```diff
@@ -11,6 +11,20 @@ class TaxonomyCountsBase():
     ]
     taxa_column = 'OTU ID'
 
+    @property
+    def rank_level(self):
```

Comment:
As parsers start with an input_file and the dataframe comes later, I couldn't put rank_level in the constructor. I started trying to code rank_level as a property only to find myself blocked by the fact that the dataframe isn't an attribute of the function. I don't really know how to go on from there... Should I establish the (temporary) dataframe ( + sep) as an attribute. Or is it starting to become too much for a little thing. Tell me what you think @khillion :)